evm/db/backends/snapshot.py

```diff
@@ -0,0 +1,9 @@
+class Snapshot(object):
+    def __init__(self, snapshot):
+        self.db = snapshot
+
+    def get(self, key):
+        return self.db.Get(key)
```

Comment:
Given the use of `self.db.Get` this object looks specific to the leveldb backend.  If that is the case, I think it should get moved into the same module as the backend class.